openassessment/xblock/test/test_openassessment.py

```diff
@@ -91,95 +95,211 @@ def test_student_view_workflow_error(self, xblock):
             xblock_fragment = self.runtime.render(xblock, "student_view")
 
         # Expect that the page renders even if the update fails
-        self.assertTrue(xblock_fragment.body_html().find("Openassessmentblock"))
+        self.assertIn("OpenAssessmentBlock", xblock_fragment.body_html())
 
-    @scenario('data/dates_scenario.xml')
```

Comment:
Oh, I actually still use that file on line 113. I just removed the decorator b/c it was acting janky with ddt.